config/initializers/devise.rb

```diff
@@ -12,7 +12,7 @@
   # Configure the e-mail address which will be shown in Devise::Mailer,
   # note that it will be overwritten if you use your own mailer class
   # with default "from" parameter.
-  config.mailer_sender = 'dahlia@housing.sfgov.org'
+  # config.mailer_sender = 'example@email.com'
```

Comment:
good catch @mluedke2, I just set it back to be a default commented out config option rather than just the "note", the comments above already explain what's going on there